test/NameLookup/members_transitive_multifile_access_level.swift

```diff
@@ -8,6 +8,9 @@
 // RUN: %target-swift-frontend -emit-module -o %t %t/MixedUses.swift
 // RUN: %target-swift-frontend -emit-module -o %t %t/InternalUsesOnlyTransitivelyImported.swift
 // RUN: %target-swift-frontend -emit-module -o %t %t/Exports.swift -I %t
+// RUN: %target-swift-frontend -emit-module -o %t %t/InternalUsesOnlySPIOnly.swift -I %t
+// RUN: %target-swift-frontend -emit-module -o %t %t/InternalUsesOnlyDefaultedImportSPIOnly.swift -I %t
+// RUN: %target-swift-frontend -emit-module -o %t %t/PublicUsesOnlySPIOnly.swift -I %t
```

Comment:
Do we have tests for the `InternalImportsByDefault` use case related to this feature?

Reply:
Not yet, but I ought to cover that case.

Reply:
Added test coverage in the latest version.